src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -128,7 +138,7 @@
             <th>{{ _('Currently on AMO') }}</th>
             <th>{{ _('Status') }}</th>
             <th>{{ _('Validation') }}</th>
-            <th class="version-delete">{{ _('Delete') }}</th>
+            <th class="version-delete">{{ _('Delete/Disable') }}</th>
```

Comment:
I think I like just "Disable" better since we're not deleting anything and it makes the wording to the user clear.